middleware/src/rpcmessages/rpcmessages.go

```diff
@@ -40,6 +40,15 @@ type SetRootPasswordArgs struct {
 	RootPassword string
 }
 
+// ToggleSetting is a generic message for settings that can be enabled or disabled
+type ToggleSetting string
+
+// Consts for toggling a setting which is done by passing "enable" or "disable" to the corresponding shell script
+const (
+	ToggleSettingENABLE  ToggleSetting = "enable"
+	ToggleSettingDISABLE ToggleSetting = "disable"
```

Comment:
Specific reason to have `ENABLE` and `DISABLE` in uppercase? I'd rather stick to the camelcase used usually in Go. e.g. `ToggleSettingEnable` and `ToggleSettingDisable`

Reply:
Yep, I'll change it. 
Just habit from BB02 where the protobuf messages are generated like that.